unstructured/ingest/connector/outlook.py

```diff
@@ -37,6 +35,7 @@ class SimpleOutlookConfig(BaseConnectorConfig):
     authority_url: Optional[str] = field(repr=False)
     ms_outlook_folders: List[str]
     recursive: bool = False
+    registry_name: str = "outlook"
```

Comment:
Could you give some explanation where/why registry name is eventually used.

Reply:
Yep, so registry_name is used so that we have a reference to get back to the original document class. By adding as a field it will end up in the serialized output of the IngestDoc. When we deserialize via the [registry](https://github.com/Unstructured-IO/unstructured/pull/1178/files#diff-d5c2c5d105fa4cf0da28254ec2fc514c1c5b6f9d5ce05167ae272611ecae6d36) `create_ingest_doc_from_json` method, we lookup the class using this name. I didn't want to just fetch/store the name of the class because this would that serialized artifacts would break if we ever refactored the name of a class (so lightly decoupling)